packages/frameworks/frameworks.json

```diff
@@ -133,10 +133,10 @@
     },
     "settings": {
       "buildCommand": {
-        "placeholder": "`npm run build` or `docusaurus-build`"
+        "placeholder": "`npm run build` or `docusaurus build`"
       },
       "devCommand": {
-        "value": "docusaurus-start --port $PORT"
+        "value": "docusaurus start --port $PORT"
```

Comment:
I think we need an additional entry here to separate 1.x and 2.x

Can you change `matchContent` to look for 1 or 2?

Reply:
It already does that